inst/qml/LSbinaryclassification.qml

```diff
@@ -46,17 +46,73 @@ Form {
 		visible: inputType.value === "uncertainEstimates" || inputType.value === "data"
 		columns: 3
 		title: qsTr("Priors")
+
 		Text{ text: qsTr("Prevalence") }
-		FormulaField { name: "priorPrevalenceAlpha";  label: qsTr("~ Beta(α = ");		afterLabel: ",";		min: 0; defaultValue: "1"	; fieldWidth: 65}
-		FormulaField { name: "priorPrevalenceBeta";   label: "β = ";								afterLabel: qsTr(")");	min: 0; defaultValue: "9"; fieldWidth: 65	}
+
+		FormulaField
```

Comment:
I think we could just set `inclusive` to false (https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-qml-guide.md#formulafield) to disallow 0 exactly...

Reply:
That's a much nicer solution. I should have read the docs more carefully.